pkg/hintrunner/utils/math_utils.go

```diff
@@ -1,21 +1,35 @@
 package utils
 
 import (
+	"fmt"
 	"math/big"
 
 	"github.com/consensys/gnark-crypto/ecc/stark-curve/fp"
 )
 
 func AsInt(valueFelt *fp.Element) *big.Int {
-	// https://github.com/starkware-libs/cairo-lang/blob/efa9648f57568aad8f8a13fbf027d2de7c63c2c0/src/starkware/cairo/common/math_utils.py#L8
-
 	var valueBig big.Int
 	valueFelt.BigInt(&valueBig)
+	return AsIntBig(&valueBig)
+}
+
+func AsIntBig(value *big.Int) *big.Int {
```

Comment:
You shouldn't return a reference to big.Int, but the value in itself. (a.k.a. `big.Int` and not `*big.Int`)

Reply:
yes thats right, changed that